### DISCUSSION TITLE
URGENT Save the final reconstructed parent grains as .ctf file

### DISCUSSION THREAD

**Discussion by EiriniGalliopoulou** [ORTools4MTEX/ORTools#49] [Q&A] [answered]
Hi to all,
I have performed parent grain reconstruction from ferrite back to austenite grains, I have obtained the cleaned plot nd now I want to save the reconstructed structure as a ctf file! Can you please tell me how can I do that?

Best,
Eirini Galliopoulou

**Comment by frankNiessen** [comment, marked as answer]
Hi, 

after parent grain reconstruction these two lines of code will do the job:

```
parentEBSD = job.calcParentEBSD
parentEBSD.export('myFile.ctf')
```

Best wishes
Frank

**Comment by EiriniGalliopoulou** [comment]
Dear Dr. Niessen,

Thank you for your answer. I was able to save the .ctf reconstructed file, but when I try to plot the ipf or KAM map of that file, I keep getting the following errors:

Not enough input arguments.

Error in isgraphics (line 3)
out = ishandle(h) && strcmpi(get(h,'type'),type);

Error in clf (line 50)
elseif ~isscalar(fig) || ~isgraphics(fig)

Error in mtexFigure (line 106)
      clf('reset');

Error in newMtexFigure (line 74)
    mtexFig = mtexFigure(varargin{:});

Error in EBSD/plot (line 58)
mtexFig = ***@***.***,ebsd},varargin{:});


Do you know why is that?

Best regards,
Eirini Galliopoulou
________________________________

**Comment by EiriniGalliopoulou** [comment]
Dear Dr. Niessen,

The problem is now resolved. Thank you for your guidance!

Best regards,
Eirini Galliopoulou
________________________________
